prow/config.yaml

```diff
@@ -11725,6 +11725,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180504-baab65e5e-master
 
+- interval: 30m
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gke-autoscaling-gpu
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=350
```

Comment:
this need to be >20min longer than config.json, aka 440 will be fine

Reply:
Thanks. I made them shorter too. 150 vs 170.